test/integration/render-tests/zoom-visibility/was-above/style.json

```diff
@@ -1,22 +1,35 @@
 {
   "version": 8,
+  "center": [
```

Comment:
Why are changes needed to this test? the unchanged test seems to work correctly even with the fix introduced in this PR.